### PR TITLE
[mongodb] allow document field defined as string type in flink

### PR DESCRIPTION
**Pull request by vanliu-tx**
Closing https://github.com/ververica/flink-cdc-connectors/issues/1384

### CODE REVIEW

**Comment by vanliu-tx** [comment]
@Jiabao-Sun @leonardBang could you help to review this?

**Review by Jiabao-Sun — COMMENTED** [review]
Thanks @vanliu-tx for contribution.
I left some comments for better readability.

File: flink-connector-mongodb-cdc/src/main/java/com/ververica/cdc/connectors/mongodb/table/MongoDBConnectorDeserializationSchema.java [hunk 1]
```diff
@@ -561,6 +561,10 @@ private StringData convertToString(BsonValue docObj) {
         if (docObj.isString()) {
             return StringData.fromString(docObj.asString().getValue());
         }
+        if (docObj.isDocument()) {
+            // convert document to json string
+            return StringData.fromString(docObj.toString());
```
[the remark this answers is not in the document]
Comment (vanliu-tx) [reply flink-connector-mongodb-cdc/src/main/java/com/ververica/cdc/connectors/mongodb/table/MongoDBConnectorDeserializationSchema.java]: thanks @Jiabao-Sun for review, have resolved this comment.

**Review by Jiabao-Sun — APPROVED** [review]
Thanks @vanliu-tx.
LGTM.

**Comment by Jiabao-Sun** [comment]
Closed #1384

**Pull request merged**